.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions/checkout@v4
       -
         name: GPG conf
-        uses: actions/github-script@v6
+        uses: actions/github-script@4020e461acd7a80762cdfff123a1fde368246fa4
```

Comment:
why are we targeting specific branch instead of tag

Reply:
it's more secure.